.circleci/continue_config.yml

```diff
@@ -340,15 +340,15 @@ workflows:
             parameters:
               engine:
                 - snowflake
-                - databricks
-                - redshift
+                #- databricks
+                #- redshift
                 - bigquery
-                - clickhouse-cloud
-                - athena
-          filters:
-           branches:
-             only:
-               - main
+                #- clickhouse-cloud
+                #- athena
+          #filters:
```

Comment:
todo: revert

note that Snowflake also inherits `ClusteredByMixin` so ive enabled Snowflake tests for this PR too